### ISSUE TITLE
Looking better

### ISSUE THREAD

**Issue by alexhultman** [Phara0h/Ouroborus#2]
Since you deleted the previous issue,

It looks a lot better now, but there are some minor nits left:

My copyright was explained as:

````
 Authored by Alex Hultman, 2018-2019.
 Intellectual property of third-party.
````

This still stands, and you cannot simplify it to just:
````
 Authored by Alex Hultman, 2018-2019.
````

There's also requirements of prominently stating the files are not original:

* You must cause any modified files to carry prominent notices stating that You changed the files; and

A simple way of doing so is to simply write a comment about what git commit it derives from and what project, or anything like that. The point is to clearly separate any derivatives from its root, while still properly attributing the root itself.

Here's an example of how another fork does it:

```
This software is based on uWebSockets 0.14 which has ZLIB license 
make sure to check /src/LICENSE.

For the code provided by cWS:

MIT License

Copyright (c) 2018 cWS

[...]
```

This is how another fork does it:

```
ZLIB License

Copyright (c) 2017 Marko Biškup

Parts of the code are based on software developed as µWebSockets,
Copyright (c) 2016 Alex Hultman and contributors.
```

You need to clearly distance yourself from the original and state changes. Other than this, go for it.

**Comment by alexhultman** [comment]
You'll also have to change the wording "Authored by Phara0h" to something accurate like "Modified by [...]". You did not author anything of those files, you modified some include guards.

**Comment by Phara0h** [comment]
> You'll also have to change the wording "Authored by Phara0h" to something accurate like "Modified by [...]". You did not author anything of those files, you modified some include guards.

Nowhere in http://www.apache.org/licenses/LICENSE-2.0 says I need to comply with that request.

**Comment by alexhultman** [comment]
Well did you originally create the work? To author something, that's to originally create something. This attribution is ridiculous.

Anyways, you're not complying with

4b:

* You must cause any modified files to carry prominent notices stating that You changed the files; and

You did not clarify that you modified the files, so it could make a lot of sense to change "Authored by" to "Modified by" so that you comply with 4b,

but you can do it any way to want to - if you really think it makes sense to write your own name as "author" to something you just copied, fine. It's a dick move but I guess it's less of an issue. Nobody is going to buy that you did, anyways.

**Comment by alexhultman** [comment]
Btw, if you want to play ugly games, there's quite a bit of source code in here that's not licensed:

All examples lack Apache License; build.c lacks license, Makefile lacks license, any many others. You'll have to remove those entirely, they are not licensed for redistribution / modification.

I can DMCA this repo solely based on those files. This game can really be played ugly if you like to.

**Comment by alexhultman** [comment]
If you look in the repo, you can see nothing is licensed in tests, benchmarks, examples, helpers, misc -> the only folder that holds any Apache License 2.0 banner is in src, so you can start by removing everything that you haven't taken from src. You have no license to redistribute anything there.

Also, you're still not complying with 4b, and still not 4c. You have cut away "Intellectual property of third party".

You can begin by fixing this, and we can talk later. This is so far a very simple case.

**Comment by Phara0h** [comment]
> Btw, if you want to play ugly games, there's quite a bit of source code in here that's not licensed:
> 
> All examples lack Apache License; build.c lacks license, Makefile lacks license, any many others. You'll have to remove those entirely, they are not licensed for redistribution / modification.
> 
> I can DMCA this repo solely based on those files. This game can really be played ugly if you like to.

Not required to be at a per-file base. Your license at the root counts for everything under it.

**Comment by alexhultman** [comment]
Nope. I am the owner of my work and I know exactly what I have licensed and not. You can even read this here:

http://www.apache.org/licenses/LICENSE-2.0#apply

There is nothing in the license saying what it applies to. If you have questions to what I have licensed, you can ask me. The author and owner of intellectual rights.

Do you know how long I've thought about this exact scenario, kid? I know all the rules and all terms of service. You're arguing about things you didn't create or decide - you can either simply follow my instructions or keep splitting hair and try to miscredit things. I can tell you now that if you play things easy, I will play it easy - I have no interest in what you do here, as long as you don't miscredit my work. Play it hard, I'll play it two times as hard. It's very simple. I currently have all the laws in the world on my side and I'll flick that DMCA button like a cherry if you refuse to comply.
